src/QuizContent.js

```diff
@@ -0,0 +1,204 @@
+    const [currentQuestionIndex, setCurrentQuestionIndex] = useState(0);
+
+    const fetchQuizData = async () => {
+        const options = {
+            method: "GET",
+            headers: 
+                {Authorization: `Bearer ${null}`}
+        };
+
+        const url = ''
+
+        try {
+            const response = await fetch(url, options)
+            if(!response.ok){
+                console.log(response);
+                const errMessage = `${response.status}`
+                throw new Error(errMessage);
+            }
+
+            const quizData = await response.json();
+
+            return quizData;
+        } catch (error) {
+            console.log(error.message);
+        }
+    }
+
+   useEffect(()=> {
+   //    const fetchQuiz = async function () {
+    //        try {
+    //            const quizQs = await fetchQuizData();
+               
+    //            const fetchedQuestions = await fetchQuizData();
+    
+    //            setQuestions(fetchedQuestions);
+    
+    //        } catch (error) {
+    //            console.error(error.message);
+    //        }
+    //    }
+        const title = "Addition Quiz"    
+
+        const quizQs = [            
+            { id: 1, question: "2+2 equals:", options: [3, 4, 5, 6], correctAnswer: 4, userAnswer: null, isCurrentQ:null },
+            { id: 2, question: "2+3 equals:", options: [3, 4, 5, 6], correctAnswer: 5, userAnswer: null, isCurrentQ:null},
+            { id: 3, question: "2+4 equals:", options: [3, 4, 5, 6], correctAnswer: 6, userAnswer: null, isCurrentQ:null},
+            { id: 4, question: "2+1 equals:", options: [3, 4, 5, 6], correctAnswer: 3, userAnswer: null, isCurrentQ:null}
+        ];
+
+        setQuestions(quizQs);
+        setQuizTitle(title);
+    }, [])
+
+    const handleAnswerSelected = (answer, resetCallback) => {
+        const selectedAnswer = parseInt(answer, 10);
+        console.log("Selected answer:", selectedAnswer);
+        const updatedQuestions = [...questions];
+        updatedQuestions[currentQuestionIndex].userAnswer = selectedAnswer;
+        setQuestions(updatedQuestions);
+
+        //changes to the next question if more questions exist
+        if(currentQuestionIndex < questions.length) {
+            setCurrentQuestionIndex(currentQuestionIndex+1);
+        }
+
+        resetCallback();
+    };
+
+    const handleSkipQuestion = (resetCallback) => {
+        //changes to the next question if more questions exist
+        if(currentQuestionIndex < questions.length-1) {
+            setCurrentQuestionIndex(currentQuestionIndex+1);
+        }
+
+        resetCallback();
+    };
+
+    const handleBackQuestion = (resetCallback) => {
+        //changes to the next question if more questions exist
+        if(currentQuestionIndex < questions.length && currentQuestionIndex > 0) {
+            setCurrentQuestionIndex(currentQuestionIndex-1);
+        }
+
+        resetCallback();
+    };
+
+    return (
+        <>
```

Comment:
You don't need an empty fragment here as you have a child as one Container element.